.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,57 @@
+name: e2e test suite
+
+on: [push]
```

Comment:
This will be changed to pushes to `master` prior to being merged in. It's currently set to on push to demonstrate it's functionality.

Reply:
So... continuous integration... but do we really need all this operating after every commit? Or only after.... ?? It's tidier to say every commit. But if you're fixing typos... don't need the tests to run. Maybe we should have a reverse flag, [NO TEST] or something like that, similar to the [DEPLOY TO DEV] in the specific site repos? ie: a way to skip the tests when it's a waste of time and resources.

Reply:
Yeah, I could definitely add a skip flag to prevent the tests from being run on a given commit. This will also evolve over time so we can see how it plays out. The ideal case in the future is to have the tests run at the PR stage and prevent merging if tests fail, but that's a future state.